web/src/app/components/bar-chart/bar-chart.component.scss

```
@@ -0,0 +1,21 @@
+.barchart{
+    height: 420px;
+    width: 1000px;
+}
```

```suggestion
.barchart {
    height: 420px;
    width: 1000px;
}
```

Same for the one below.